temp1.js

```diff
@@ -0,0 +1,110 @@
+const {
+    Worker, isMainThread, workerData, BroadcastChannel, MessageChannel,
+  } = require('worker_threads');
+  const CoinbasePro = require('coinbase-pro');
+  const stringTable = require('string-table');
+  const moment = require('moment');
+  const log4js = require('log4js');
+  const { v4 } = require('uuid');
+  const { wsUrl } = require('./model/constants');
+  const { strategies } = require('./strategies/all_strategies');
+  const { loadConfigurationFile } = require('./utils/loadConfigurationFile');
+  const { checkEnvironmentVariables } = require('./utils/checkEnvironmentVariables');
+  const botConfiguration = loadConfigurationFile(process.argv);
+  const tickerChannel = new BroadcastChannel('ticker');
+  const candleChannel = new BroadcastChannel('candles-every-minute-past-10-minutes');
+  const logAppenders = {};
+  const appendersName = ['main'];
+  const logCategories = {};
+  strategies.forEach((strategy) => {
+    const identifier = v4();
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/amcalabretta/botbase/pullRequest?prid=7628479)
```suggestion
  const identifier = v4();
```